sentinelsat/sentinel.py

```diff
@@ -117,9 +117,12 @@ class SentinelAPI(object):
         Session to connect to DataHub
     api_url : str
         URL to the DataHub
+    page_size : int
+        number of results per query page
+        current value: 100 (maximum allowed on ApiHub)
     """
 
-    def __init__(self, user, password, api_url='https://scihub.copernicus.eu/apihub/', max_rows=100):
+    def __init__(self, user, password, api_url='https://scihub.copernicus.eu/apihub/', page_size=100):
```

Comment:
Setting the page_size on init is redundant due to L134 (setting page sized as fixed attribute).